src/main/java/frc/robot/commands/teleopcommands/ClimberUp.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.teleopcommands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.OI;
+import frc.robot.RobotMap;
+
+public class ClimberUp extends CommandBase {
+  /**
+   * Creates a new ClimberUp.
+   */
+  public static double startTime;
+
+  public ClimberUp() {
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    startTime = Timer.getFPGATimestamp();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    RobotMap.climberRelease.set(Value.kForward);
+    if(Timer.getFPGATimestamp() - startTime > 1 && Timer.getFPGATimestamp() - startTime < 4) {
```

Comment:
Take all these constants and at least put them in `private static final double MY_DESCRIPTIVE_CONSTANT_NAME = 0.4;` at the top of this class